templates/report.js

```diff
@@ -2,4 +2,12 @@ angular.module('app', [])
   .controller('ReportController', function($scope, $locale) {
     $scope.results = results;
     $scope.jsonData = JSON.stringify(results, null, 4);
+    $scope.suite = results.suites.length;
+    $scope.data = results.suites.reduce(function(suiteTot, suite) {
+      var specCount = suite.specs.reduce(function(specTot, spec) {
+        return {fail: specTot.fail + spec.failedExpectations.length, pass: specTot.pass + spec.passedExpectations.length};
+      }, {fail:0, pass:0})
+
+      return {specs: suite.specs.length, fail: suiteTot.fail + specCount.fail, pass: suiteTot.pass + specCount.pass};
```

Comment:
you are initializing with test, fail and pass, but here returning specs, fail & pass ?

Reply:
need to change the returning variable name to test??